src/CommunityToolkit.Maui.SourceGenerators.Internal/Range.cs

```diff
@@ -0,0 +1,242 @@
+﻿// https://github.com/dotnet/runtime/blob/419e949d258ecee4c40a460fb09c66d974229623/src/libraries/System.Private.CoreLib/src/System/Index.cs
+// https://github.com/dotnet/runtime/blob/419e949d258ecee4c40a460fb09c66d974229623/src/libraries/System.Private.CoreLib/src/System/Range.cs
+
+using System.Runtime.CompilerServices;
+
+namespace System
```

Comment:
wouldn't be better to use this lib instead?
 - https://github.com/Sergio0694/PolySharp